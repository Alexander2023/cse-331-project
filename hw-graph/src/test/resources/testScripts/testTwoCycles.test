# Create a graph
CreateGraph graph1

# Add a single node
AddNode graph1 n1

# Add two edges to self with different labels
AddEdge graph1 n1 n1 e1
AddEdge graph1 n1 n1 e2

# Print the children of single node
ListChildren graph1 n1