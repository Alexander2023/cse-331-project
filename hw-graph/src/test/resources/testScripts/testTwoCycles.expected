# Create a graph
created graph graph1

# Add a single node
added node n1 to graph1

# Add two edges to self with different labels
added edge e1 from n1 to n1 in graph1
added edge e2 from n1 to n1 in graph1

# Print the children of single node
the children of n1 in graph1 are: n1(e1) n1(e2)